test/utils/test_filesystem.py

```diff
@@ -53,3 +56,29 @@ def test_http_package():
 
     known_checksum = "e075b2836d03f7971f754354807dcdca51a7875c8297cb161557946736d1f7fc"
     assert calculate_checksum(path) == known_checksum
+
+
+@pytest.mark.skip("Skipping because slow, need better test solution")
```

Comment:
Thanks for adding this, but not sure if we should skip. If looking for a smaller file, this [MGN checkpoint](https://catalog.ngc.nvidia.com/orgs/nvidia/teams/modulus/models/modulus_ns_meshgraphnet) is ~50% of the DLWP checkpoint size and might be worth a try.

Reply:
Yeah I'm waiting on the diagnostic models to get uploaded, I can use climatenet checkpoint for this test which is like 2Mb download.

Will have a follow up PR when they are live.